### PR TITLE
Pursuance EGO

### PR DESCRIPTION
**Pull request by selectL-L**
Hopefully the copy and paste job actually manages to get the EGO to work properly in the calculator, if the image needs updating too I can modify it, I still have the file on my PC.

Thanks in advance!

### CODE REVIEW

File: public/egos.json [hunk 1]
```diff
@@ -846,6 +846,23 @@
...
+    "type": "Slash",
+    "costs": [
+      {
+        "affinity": "Sloth",
+        "cost": 4
+      },
+      {
+        "affinity": "Pride",
+        "cost": 3
+      },
```
Comment (darylhcw) [inline public/egos.json]: Please remove trailing comma here.
The default JSON parser follows the JSON spec and errors out on this.

Comment (darylhcw) [reply public/egos.json]: Everything else is good!

Comment (selectL-L) [reply public/egos.json]: Went ahead and fixed the trailing comma!

Sorry for mucking up a little on that

**Comment by darylhcw** [comment]
Thanks again for the help!

**Pull request merged**